packages/core/theme-module/theme/pages/Category.vue

```diff
@@ -375,15 +376,13 @@ export default {
   setup(props, context) {
     const { params, query } = context.root.$route;
 
-    const lastSlug = Object.keys(params).reduce((prev, curr) => params[curr] || prev, params.slug_1);
-
     const { categories, search, loading } = useCategory('categories');
     const { products: categoryProducts, totalProducts, search: productsSearch, loading: productsLoading } = useProduct('categoryProducts');
     const currentPage = ref(parseInt(query.page, 10) || 1);
     const itemsPerPage = ref(parseInt(query.items, 10) || perPageOptions[0]);
 
     onSSR(async () => {
-      await search({ slug: lastSlug });
+      await search(getCategorySearchParameters(context));
```

Comment:
I reminded you that it should be inlined. `includes` dir should exist only in `theme-module`

Reply:
We agreed that in the end, we will move this one to an external function